tests/test_interrupted_sync.py

```diff
@@ -0,0 +1,144 @@
+from base import DixaBaseTest
+from tap_tester.logger import LOGGER
+
+
+class DixaInterruptedSyncTest(DixaBaseTest):
+    """Test tap sets a bookmark and respects it for the next sync of a stream"""
+
+    def name(self):
+        return "tap_tester_dixa_bookmark_test"
+
+    def test_run(self):
+        """
+        - Verify an interrupted sync can resume based on the currently_syncing and stream level bookmark value.
+        - Verify only records with replication-key values greater than or equal to the stream level bookmark 
+          are replicated on the resuming sync for the interrupted stream.
+        - Verify the yet-to-be-synced streams are replicated following the interrupted stream in the resuming sync. 
+          (All yet-to-be-synced streams must replicate before streams that were already synced)
+        """
+        
+        expected_streams = self.expected_streams()
+        expected_replication_keys = self.expected_replication_keys()
+        expected_replication_methods = self.expected_replication_method()
+
+        ##########################################################################
+        # First Sync
+        ##########################################################################
+        conn_id = connections.ensure_connection(self)
+
+        # Run in check mode
+        found_catalogs = self.run_and_verify_check_mode(conn_id)
+
+        # table and field selection
+        catalog_entries = [catalog for catalog in found_catalogs
+                           if catalog.get('tap_stream_id') in expected_streams]
+        
+        self.perform_and_verify_table_and_field_selection(
+            conn_id, catalog_entries)
+
+        # Run a first sync job using orchestrator
+        first_sync_record_count = self.run_and_verify_sync(conn_id)
+        first_sync_records = runner.get_records_from_target_output()
+        first_sync_bookmarks = menagerie.get_state(conn_id)
+
+        ##########################################################################
+        # Update State Between Syncs
+        ##########################################################################
+        
+        LOGGER.info("Current Bookmark: {}".format(first_sync_bookmarks))
+        interrupted_sync_states = { 'currently_syncing': 'messages',
+                                    'bookmarks': {  'conversations': first_sync_bookmarks.get('bookmarks', {}).get('conversations', {}),
+                                                    'messages': {list(expected_replication_keys['messages'])[0]: 1655856000000},
+                                                    'activity_logs': {list(expected_replication_keys['activity_logs'])[0]: self.start_date}}}        
+        completed_streams = {'conversations'}
+        pending_streams = {'activity_logs'}
+
+        menagerie.set_state(conn_id, interrupted_sync_states)
+        LOGGER.info("New Bookmark: {}".format(menagerie.get_state(conn_id)))
```

Comment:
I think this gets logged by the tap-tester backend in set_state. So should be redundant to log here.